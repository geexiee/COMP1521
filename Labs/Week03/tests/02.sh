./maf 1 10000000 01000000000000000000000
