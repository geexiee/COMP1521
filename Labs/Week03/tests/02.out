bits : 1 10000000 01000000000000000000000
float: -2.5000000000
